engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/views/product-repositories.html

```diff
@@ -1,6 +1,14 @@
 <span page-title ng-model="product">{{ 'Repositories for Product:' | translate }} {{ product.name }}</span>
 
 <section class="details details-full">
+  <div bst-alert="success" ng-show="removingTasks.length != 0" close="close($index)" ng-repeat="task in removingTasks">
```

Comment:
This may be nitpicking but I'd say to use `ng-show="removingTasks.length > 0"` or `ng-hide="removingTasks.length == 0"`